Traceback (most recent call last):
  File "/home/gridsan/tloman/.local/lib/python3.9/site-packages/gillespy2/core/model.py", line 305, in _resolve_parameter
    parameter.validate()
  File "/home/gridsan/tloman/.local/lib/python3.9/site-packages/gillespy2/core/parameter.py", line 124, in validate
    raise ParameterError("expression must be of type str, float, or int.")
gillespy2.core.gillespyError.ParameterError: expression must be of type str, float, or int.

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/gridsan/tloman/.local/lib/python3.9/site-packages/gillespy2/core/model.py", line 1295, in run
    return solver.run(t=t, increment=increment, timeout=timeout, **solver_args)
  File "/home/gridsan/tloman/.local/lib/python3.9/site-packages/gillespy2/solvers/numpy/tau_hybrid_solver.py", line 1012, in run
    self.model.compile_prep()
  File "/home/gridsan/tloman/.local/lib/python3.9/site-packages/gillespy2/core/model.py", line 1232, in compile_prep
    self._resolve_all_parameters()
  File "/home/gridsan/tloman/.local/lib/python3.9/site-packages/gillespy2/core/model.py", line 315, in _resolve_all_parameters
    self._resolve_parameter(parameter)
  File "/home/gridsan/tloman/.local/lib/python3.9/site-packages/gillespy2/core/model.py", line 311, in _resolve_parameter
    raise ModelError(f"Could not add/resolve parameter: {parameter.name}, Reason given: {err}") from err
gillespy2.core.gillespyError.ModelError: Could not add/resolve parameter: BT, Reason given: expression must be of type str, float, or int.

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/gridsan/tloman/Projects/Benchmarks/GillesPy2/gillespy2_benchmark_protptyping.py", line 102, in <module>
    tc_BCR_THS = model_BCR.run(solver=TauHybridSolver,t=bcr_ss_time,increment=bcr_ss_time/200.0)
  File "/home/gridsan/tloman/.local/lib/python3.9/site-packages/gillespy2/core/model.py", line 1297, in run
    raise SimulationError(
gillespy2.core.gillespyError.SimulationError: argument 'solver=<gillespy2.solvers.numpy.tau_hybrid_solver.TauHybridSolver object at 0x7ff245621f70>' to run() failed.  Reason Given: Could not add/resolve parameter: BT, Reason given: expression must be of type str, float, or int.

real	0m50.474s
user	0m41.124s
sys	0m8.791s
